packages/sit-onyx/src/components/OnyxRadioButtonGroup/OnyxRadioButtonGroup.vue

```diff
@@ -0,0 +1,42 @@
+<script lang="ts" setup generic="T">
+import OnyxRadioButton from "../OnyxRadioButton/OnyxRadioButton.vue";
+import type { SelectionOption } from "../OnyxRadioButton/types";
+
+const props = defineProps<{
+  /**
+   * name for the radio buttons
+   */
+  name: string;
+  modelValue?: SelectionOption<T>;
+  label?: string;
+  isDisabled?: boolean;
+  isReadonly?: boolean;
+  isLoading?: boolean;
+  options: SelectionOption<T>[];
+}>();
+
+const emit = defineEmits<{
+  "update:modelValue": [SelectionOption<T>];
+}>();
+</script>
+
+<!-- TODO: loading -->
+<!-- TODO: readonly -->
+<template>
+  <fieldset class="onyx-radio-button-group">
+    <legend v-if="props.label" class="onyx-radio-button-group__legend">{{ props.label }}</legend>
```

Comment:
```suggestion
    <legend v-if="props.label" class="onyx-radio-button-group__legend">{{ props.label }}</legend>
    <!-- TODO: required "*" on legend -->
```

Reply:
Styling follows in the upcoming PRs, I don't want to add a TODO for ever small thing. We have the accpetance criteria for that :)

Reply:
ah right, it's done via CSS. sry ^^